.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,57 @@
+// /*
+//  * Copyright contributors to the Galasa project
+//  *
+//  * SPDX-License-Identifier: EPL-2.0
+//  */
+ 
+{
+  "name": "Java",
+	"image": "mcr.microsoft.com/devcontainers/java:17",
+  
+  "features": {
+    "ghcr.io/devcontainers/features/java:1": {
+      "jdkDistro": "sem",
+      "version": "17.0.14-sem",
+			"installMaven": "true",
+			"mavenVersion": "3.9.0",
+			"installGradle": "true",
+			"gradleVersion": "8.14.3"
+    },
+    
+    "ghcr.io/devcontainers/features/python:1": {
+			"version": "3.11.0"
+    },
+    
+    "ghcr.io/devcontainers/features/go:1": {
+			"version": "1.23.5"
+    },
+    
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": "20.10.0"
+    },
+
+    "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {
+			"version": "28.3.3-1",
+      "moby": false
+		}
+  },
```

Comment:
If someone doesn't have docker installed outside of the dev container, would the docker-outside-of-docker feature cause issues?